lambda/cloudwatchAlarm/index.js

```diff
@@ -1,6 +1,7 @@
 const axios = require('axios');
-const { utcToZonedTime } = require('date-fns-tz');
-const { timeZone } = require('../dynamoUtil');
```

Comment:
Do we still need timeZone and TIMEZONE in dynamoUtil? i.e. Would they ever be different?

Reply:
No, that looks to be a duplicate in `dynamoUtil.js`. I can remove it after my current PRs are merged since the state of the code at the moment relies on both being available.